package.json

```diff
@@ -2,14 +2,23 @@
   "name": "headers-polyfill",
   "version": "3.0.10",
   "description": "A native \"Headers\" class polyfill.",
-  "main": "lib/index.js",
-  "types": "lib",
+  "main": "./lib/index.js",
+  "module": "./lib/esm/index.js",
+  "types": "./lib/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
```

Comment:
we might eventually add a `browser` export as well here, that just re-exports from `globalThis` instead of always using the polyfill, since browsers _should_ all support `Headers` now, but that's outside the scope of this work

Reply:
@mattcosta7, this may be a good idea indeed for the future improvements.